plugins/module_utils/common/json_to_xml.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
```

Comment:
SInce this is utils code not used by module it can be moved under `plugins_utils` folder

Reply:
ack